projects/elife.yaml

```diff
@@ -947,82 +947,41 @@ iiif:
         type: t2.medium
         ports:
             - 22
-            - 80
-            - 443
+            - 80:
+                cidr-ip: 10.0.0.0/16 # ELB only
+        ec2:
+            cluster-size: 2
+        elb:
+            protocol: 
+                - https
+            healthcheck:
+                path: /
+        cloudfront:
```

Comment:
not sure `cloudfront` should be needed as a default: `ci` for example doesn't need it

Reply:
the intent is to show the normal case for this project, not the minimal. the CI configuration is the odd one out looking at all the duplication.